internal/ui/ui_test.go

```diff
@@ -0,0 +1,45 @@
+package ui
+
+import (
+	"testing"
+)
+
+func TestConcatStringsSingleString(t *testing.T) {
```

Comment:
Looping through input, expected tuples and doing assert in a sub-test would also work, as all tests have the same check, but no big difference.